docker/cluster-health-monitor.Dockerfile

```diff
@@ -10,18 +10,23 @@ COPY go.sum go.sum
 RUN go mod download
 
 # Copy the go source
-COPY cmd/clusterhealthmonitor/ cmd/clusterhealthmonitor/
+COPY cmd/ cmd/
 COPY pkg/ pkg/
+COPY apis/ apis/
 
 # Build
 RUN CGO_ENABLED=1 GOEXPERIMENT=systemcrypto go build -o clusterhealthmonitor cmd/clusterhealthmonitor/main.go
```

Comment:
I think these components are meant to always deploy together so it's probably fine to keep it all in one Dockerfile/image for now. That said, the `ENTRYPOINT ["/clusterhealthmonitor"]` line should probably be removed now that there are multiple images and we expect the binary that will be run to be specified in the k8s manifests. Maybe we can also add a comment stating this.

Reply:
I added a TODO on the Dockerfile.

Reply:
Thanks